spec/models/participant_outcome/npq_spec.rb

```diff
@@ -231,4 +231,135 @@
       end
     end
   end
+
+  describe "#has_failed?" do
+    it "returns true if failed" do
+      outcome = described_class.new(state: "failed")
+      expect(outcome.has_failed?).to eql(true)
+    end
+
+    it "returns false if passed" do
+      outcome = described_class.new(state: "passed")
+      expect(outcome.has_failed?).to eql(false)
+    end
+
+    it "returns nil if voided" do
+      outcome = described_class.new(state: "voided")
+      expect(outcome.has_failed?).to eql(nil)
+    end
+  end
+
+  describe "#passed_and_not_sent?" do
+    it "returns true if passed and not sent" do
+      outcome = described_class.new(state: "passed", sent_to_qualified_teachers_api_at: nil)
+      expect(outcome.passed_and_not_sent?).to eql(true)
+    end
+
+    it "returns false if passed and sent" do
+      outcome = described_class.new(state: "passed", sent_to_qualified_teachers_api_at: Time.zone.now)
+      expect(outcome.passed_and_not_sent?).to eql(false)
+    end
+
+    it "returns false if failed" do
```

Comment:
Would it be worth setting not sent explicitly in these scenarios to make sure its only testing the failed status?

```
outcome = described_class.new(state: "failed", sent_to_qualified_teachers_api_at: nil)
```